core/location/db_resolver.go

```diff
@@ -57,18 +57,22 @@ func (r *DBResolver) DetectLocation() (Location, error) {
 
 // ResolveLocation maps given ip to country.
 func (r *DBResolver) ResolveLocation(ip net.IP) (Location, error) {
```

Comment:
We can do it a bit smaller with a named returned vars: 
```suggestion
func (r *DBResolver) ResolveLocation(ip net.IP) (loc Location, err error) {
```